src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -22,19 +24,20 @@ object BooleanOperatorsSpecification extends Properties("Boolean Operators"):
 
   property("and") = forAll { (pair: (Boolean, Boolean)) =>
```

Comment:
Please add extra test for the case to check that `right` is not calculated.
Hint: 

- since all parameters are know, use `propBoolean` instead of `forAll`;
- throw an error as `right` argument.